pkgs/development/python-modules/qds_sdk/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib,
+ fetchPypi,
+ buildPythonPackage,
+ boto,
+ inflection,
+ pytest,
+ requests,
+ six,
+ urllib3 }:
+
+buildPythonPackage rec {
+  pname = "qds_sdk";
+  version = "1.12.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "qds_sdk";
+    sha256 = "113qrfz165sqcbndgdpvr4q7p08bdfw53sh1wrkvip8g2cngh6p3";
+  };
+
+  # No tests available
+  doCheck = true;
```

Comment:
since they don't distribute tests with their sdist pypi package, we could also pull from github: https://github.com/qubole/qds-sdk-py.

Tests are pretty import to ensure the the package isn't going to be broken when upstream dependencies get bumped :)

Reply:
Thank you for the review, I have updated the code to use `fetchgit` and run the tests.

Reply:
Resolving this as dupe of https://github.com/NixOS/nixpkgs/pull/70055#discussion_r330065091.